app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -146,22 +150,26 @@ export default {
     async submitFormSchema() {
       if (this.$refs.form.validate()) {
         if (this.formId && this.formVersionId) {
+          // console.info('submitFormSchema(1)'); // eslint-disable-line no-console
           // If editing a form, update the version
           try {
             const response = await formService.updateVersion(
               this.formId,
               this.formVersionId,
               {
-                schema: formSchema,
+                schema: this.formSchema,
               }
             );
             const data = response.data;
-            formSchema = data.schema;
+            // console.info(`response.data.schema = ${JSON.stringify(data.schema)}`); // eslint-disable-line no-console
```

Comment:
Want to remove all the commented out console logs or leave them?

Reply:
I think we should clear them out. We can always add them back in when we need to debug it next time.

Reply:
Left them in so you guys could know what was happening. Will remove them now.